src/views/tables/OrdersTable/OrderActionsCell.tsx

```diff
@@ -48,22 +53,24 @@ export const OrderActionsCell = ({ orderId, orderFlags, status, isDisabled }: El
 
   return (
     <ActionsTableCell>
-      <$CancelButton
-        key="cancelorder"
-        iconName={IconName.Close}
-        iconSize="0.875em"
-        shape={ButtonShape.Square}
-        $uiRefreshEnabled={uiRefresh}
-        {...(isOrderStatusClearable(status)
-          ? { onClick: () => dispatch(clearOrder(orderId)) }
-          : {
-              onClick: onCancel,
-              state: {
-                isLoading: isCanceling,
-                isDisabled: isCancelDisabled,
-              },
-            })}
-      />
+      <WithTooltip tooltipString={stringGetter({ key: STRING_KEYS.CANCEL_ORDER })}>
```

Comment:
The button also acts as "dismiss" so I think we'll need a conditional here / different tooltip!

Reply:
ooh ur right - updated to "clear" when applicable

<img width="1210" alt="Screenshot 2024-10-18 at 6 59 01 PM" src="https://github.com/user-attachments/assets/afe099e1-fc89-4bdc-a65b-cc6936eb14ed">

<img width="1209" alt="Screenshot 2024-10-18 at 7 01 22 PM" src="https://github.com/user-attachments/assets/67bb5295-e5b6-40fc-83f2-910649265402">